images/pod/Dockerfile

```diff
@@ -1,16 +1,22 @@
 #
-# This is the official OpenShift Origin pod infrastructure image. It will stay running
+# This is the official OpenShift pod infrastructure image. It will stay running
 # until terminated by a signal and is the heart of each running pod. It holds on to
 # the network and IPC namespaces as containers come and go during the lifetime of the
 # pod.
 #
 # The standard name for this image is openshift/origin-pod
 #
-FROM scratch
+FROM openshift/origin-source
```

Comment:
It seems that this has already changed for OSBS images (they are using rhel7.3). Doesn't this mean that every pod will run with the additional size of a rhel image on top of the application image? :)